src/core/core.tooltip.js

```diff
@@ -790,7 +790,7 @@ class Tooltip extends Element {
 			lines = bodyItem.lines;
 			for (j = 0, jlen = lines.length; j < jlen; ++j) {
 				// Draw Legend-like boxes if needed
-				if (displayColors) {
+				if (displayColors && j === 0) {
```

Comment:
How about doing it outside the loop instead?

```
if (displayColors) {
	me._drawColorBox(ctx, pt, i, rtlHelper);
}
for (j = 0, jlen = lines.length; j < jlen; ++j) {
	fillLineOfText(lines[j]);
}
```

Reply:
Done. I had to modify the check for the draw color check, since it's only done if there are lines of text.